traffic_server/README.md

```diff
@@ -0,0 +1,88 @@
+# Agent Check: Traffic Server
+
+## Overview
+
+This check monitors [Traffic Server][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.
+
+### Installation
+
+The Traffic Server check is included in the [Datadog Agent][2] package.
+
+To enable monitoring in Traffic Server, enable the [Stats Over HTTP plugin][10] on your Traffic Server by adding the following line to your `plugin.config` file:
+
+```
+stats_over_http.so
+```
+
+### Configuration
+
+1. Edit the `traffic_server.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your Traffic Server performance data. See the [sample traffic_server.d/conf.yaml][4] for all available configuration options.
+
+2. [Restart the Agent][5].
+
+### Validation
+
+[Run the Agent's status subcommand][6] and look for `traffic_server` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][7] for a list of metrics provided by this check.
```

Comment:
```suggestion
See [metadata.csv][7] for a list of metrics provided by this integration.
```
This phrase will be automatically replaced by the actual metric table. To stay on the safe side, it should exactly match the "magic phrase" used in the other README files.

Reply:
Is this a new requirement? I can update the new integration template if so

Reply:
No, it is not a new requirement, but yes, please update the new integration template.